libc-test/build.rs

```diff
@@ -5593,6 +5593,21 @@ fn test_aix(target: &str) {
         // values because non-unique values are being used which will
         // fail the test when _ALL_SOURCE is defined.
         "EWOULDBLOCK" | "ENOTEMPTY" => true,
+
+        // These constants are intended for use as the 'int request' argument to 'ioctl()'.
```

Comment:
Could you make this comment `FIXME(ctest)`? Because if it's an issue that only shows up during tests but not in C, presumably we could do some better validation on our end.